install_themes/action.yml

```diff
@@ -54,3 +58,53 @@ runs:
             ${{ inputs.container }} \
             composer update ${{ inputs.update_options }}
         fi
+
+    - name: Generate deprecation warning to summary page
```

Comment:
This does not generate anything on the "Summary" page but inside the job output. See https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/ for more info